### ISSUE TITLE
7.7.0: Missing dependency in lib/hx509

### ISSUE THREAD

**Issue by 0-wiz-0** [heimdal/heimdal#851]
There seems to be a missing dependency in lib/hx509 in the 7.7.0. release.

Here's the bug report against pkgsrc: https://mail-index.netbsd.org/pkgsrc-users/2021/11/04/msg034663.html

```
In file included from hxtool.c:34:0:
hx_locl.h:66:10: fatal error: ocsp_asn1.h: No such file or directory
 #include <ocsp_asn1.h>
          ^~~~~~~~~~~~~
compilation terminated.
*** [hxtool.o] Error code 1
```

A workaround for the generated file `Makefile.in` was committed to pkgsrc that fixes this issue:
```
Wait for generated includes.

--- lib/hx509/Makefile.in.orig 2021-11-04 17:38:39.000000000 +0000
+++ lib/hx509/Makefile.in
@@ -218,6 +218,7 @@ nodist_hxtool_OBJECTS = hxtool-commands.
 hxtool_OBJECTS = $(dist_hxtool_OBJECTS) $(nodist_hxtool_OBJECTS)
 hxtool_DEPENDENCIES = libhx509.la $(top_builddir)/lib/asn1/libasn1.la \
       $(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
+      $(nodist_include_HEADERS) \
       $(top_builddir)/lib/sl/libsl.la
 test_expr_SOURCES = test_expr.c
 test_expr_OBJECTS = test_expr.$(OBJEXT)
```

Please fix this. Thanks!

**Comment by jaltman** [comment]
Please provide the author information for the pkgsrc patch

**Comment by 0-wiz-0** [comment]
The patch was committed to pkgsrc by Adam Ciarcinski <adam@NetBSD.org>.

**Comment by jaltman** [comment]
The provided patch is against the Makefile.in file which is a generated file created by automake.

This issue was fixed on the master branch by 572a6fd7ac41e9210ef3eb765fe7da4ec8a94bb2 and fixed on heimdal-7-1-branch by c4cff6859d183a40fb35a76e2bc1ce084b3a6d67.

I'm closing this issue.

**Comment by 0-wiz-0** [comment]
Thank you, @jaltman !
